backend/integration_tests/src/communities/send_message_tests.rs

```diff
@@ -1,14 +1,17 @@
+use crate::client::icrc1::happy_path;
```

Comment:
Seems strange for `happy_path` within a communities test to be the `icrc1::happy_path`